File: src/main/java/plus/maa/backend/controller/request/user/LoginDTO.java
package plus.maa.backend.controller.request.user;

import jakarta.validation.constraints.Email;
import jakarta.validation.constraints.NotBlank;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.Accessors;

/**
 * @author AnselYuki
 */
@Data
@Accessors(chain = true)
@NoArgsConstructor
@AllArgsConstructor
public class LoginDTO {
    @Email(message = "邮箱格式错误")
    private String email;
    @NotBlank(message = "请输入用户密码")
    private String password;
}

File: src/main/java/plus/maa/backend/controller/request/user/LoginDTO.kt
package plus.maa.backend.controller.request.user

import jakarta.validation.constraints.Email
import jakarta.validation.constraints.NotBlank

/**
 * @author AnselYuki
 */
data class LoginDTO(
    @field:NotBlank(message = "邮箱格式错误")
    @field:Email(message = "邮箱格式错误")
    val email: String,
    @field:NotBlank(message = "请输入用户密码")
    val password: String
)
